class-tgm-plugin-activation.php

```diff
@@ -2095,7 +2095,7 @@ protected function _gather_plugin_data() {
 				if ( ! empty( $upgrade_notice ) ) {
 					$table_data[ $i ]['upgrade_notice'] = $upgrade_notice;
 
-					add_action( "tgmpa_after_plugin_row_$slug", array( $this, 'wp_plugin_update_row' ), 10, 2 );
+					add_action( 'tgmpa_after_plugin_row_' . $slug, array( $this, 'wp_plugin_update_row' ), 10, 2 );
```

Comment:
Will do.
Though WP Parser should be fixed in that case - I believe my WP Hook Documentor can actually deal with both ;-)
Re: Scrutinizer - we can probably just set them to ignore in their interface for these lines without changing the config.